protocol/p2p/src/core/router.rs

```diff
@@ -102,6 +102,14 @@ struct RouterMutableState {
 
     /// Duration of the last ping to this peer
     last_ping_duration: u64,
+
+    /// Time of the last block transfer
+    /// This corresponds to the last full block transfer, not the last Block Invoice message.
```

Comment:
`inv` stands for `inventory` and not `invoice`. Nevertheless, I think it's better to just say "Block Inv"

Reply:
done